code/modules/mob/living/carbon/alien/alien_say.dm

```diff
@@ -14,8 +17,7 @@
 			to_chat(S, "[link] [rendered]")
 
 /mob/living/carbon/alien/humanoid/royal/queen/alien_talk(message, shown_name = name)
-	shown_name = "<FONT size = 3>[shown_name]</FONT>"
-	..(message, shown_name)
+	..(message, shown_name, TRUE)

```

Comment:
would it be possible to give alien queens a special hivenode subtype, then tie speaking in beeg text over hivemind chat to that organ? that way, pseudoqueens (humans that have had all of the internal organs (bar the brain) of an alien queen transplanted into them) won't be drowned out by normal hivemind chat.

Reply:
I'm sure it could be possible, but I don't think I am ready to tackle something of that scope just yet. Makes sense if they are filling the role of an actual queen laying eggs and defending the nest the voice should be included.

My initial thought for this was just "larva need a memory about :a for hivemind, and why doesn't the bad-ass alien queen get big text like a blob overmind."

Reply:
fair enough